### ISSUE TITLE
[Taskbar auto-hide when maximized] Enable new show/hide animation

### ISSUE THREAD

**Issue by Giggigx** [ramensoftware/windhawk-mods#1236]
It would be cool to implement a toggle that enables globally the new animation that happens exclusively when you fullscreen an UWP app. 

**Normal animation**

https://github.com/user-attachments/assets/d5a49dbc-4756-4c32-8255-0c9a9649a933

**New animation**

https://github.com/user-attachments/assets/6cacb61d-daca-45a0-874b-47da7743e876

**Comment by MuscularPuky** [comment]
Because Hide Taskbar (legacy) and Fullscreen (modern) are different. as I know there's no option to control new taskbar animation you mentioned, so it'll be solutoin for it to disable auto hide (legacy) while Fullscreen. then immersive mode (modern)'ll hide the taskbar with ease

**Comment by MuscularPuky** [comment]
> Perhaps tweaking the implementation of the native tablet taskbar could help. For instance, navigating to "**HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer**", creating a new **DWORD (32-bit)** value named **TabletPostureTaskbar**, and setting it to **1** will keep the taskbar at its normal size (preventing it from becoming larger). The animation works as expected, but the system tray icons, along with the date and time, are cut off. If the icons could be hidden until the taskbar is brought up, it would resolve this issue.
> 
> Also, the taskbar doesn't overlay on top of maximized windows unless you restart explorer.exe. However, once it's restarted, the taskbar becomes much larger, which looks awkward on a monitor. Therefore, maybe tweaking the tablet taskbar size (if possible) seems like the best approach to addressing this problem? Mentioning @m417z, since someone on Reddit requested that this feature be added to Windhawk.

yes that was me https://github.com/ramensoftware/windhawk-mods/issues/593

**Comment by m417z** [comment]
That's not the new animation, but this new mod allows to customize the regular auto-hide animation: [Taskbar auto-hide speed](https://windhawk.net/mods/taskbar-auto-hide-speed).

**Comment by m417z** [comment]
That's [being worked on by Microsoft](https://www.reddit.com/r/Windows11/comments/1oja070/new_hide_taskbar_animation_is_incredible/), and can be enabled in newer/preview builds with ViVeTool ID 41356296. I have no plans to work on it for older Windows versions.
